src/utils/Card/index.examples.js

```diff
@@ -3,11 +3,6 @@ import {storiesOf} from '@kadira/storybook'
 import CourseCard from './CourseCard'
 import LessonCard from './LessonCard'
 import PlaylistCard from './PlaylistCard'
-import Card from './'
-import imgCourseCard from './assets/temp/img-course-card.png'
-import imgRx from './assets/temp/rx.svg'
-import imgAngular from './assets/temp/angular.svg'
-import imgJs from './assets/temp/js.svg'
```

Comment:
Make sure to remove all these static files when done implementing. Both the imports and the source files.